frontend/src/components/BeforeUser.tsx

```diff
@@ -0,0 +1,73 @@
+import { Col, Row } from 'react-bootstrap';
+import { Link } from 'react-router-dom';
+
+import Footer from '@components/Footer';
+
+interface Props {
+  imgUrl: string;
+  title: string;
+  content: string;
+  subContent: string;
+  way: string;
+  path: string;
+  url: string;
+}
+
+interface Data {
+  data: Props | undefined;
+}
```

Comment:
if you are unsure about the types of your input, you probably should use [zod](https://github.com/colinhacks/zod) and proper error handling to validate your data, but this could be done in another PR

Reply:
i was fixing something before, but now i can get rid of the undefined.
but using zod sounds a good idea